guides/common/modules/proc_configuring-project-to-use-ldap.adoc

```diff
@@ -31,8 +31,8 @@ Selected organizations are assigned to users created from the LDAP authenticatio
 . Click *Submit*.
 . Configure new accounts for LDAP users:
 +
-* If you did not select *Automatically Create Accounts In {Project}* checkbox, see xref:Creating_a_User_{context}[] to create user accounts manually.
+* If you did not select *Automatically Create Accounts In {Project}* checkbox, see {AdministeringDocURL}Creating_a_User_{project-context}[Creating a User] in _{AdministeringDocTitle}_ to create user accounts manually.
```

Comment:
```suggestion
* If you did not select *Automatically Create Accounts In {Project}* checkbox, see {AdministeringDocURL}Creating_a_User_admin[Creating a User] in _{AdministeringDocTitle}_ to create user accounts manually.
```

See https://github.com/theforeman/foreman-documentation/blame/master/guides/doc-Administering_Project/master.adoc#L3 or run `git grep "{AdministeringDocURL}"` to see some examples.

Reply:
Well I am confused. In your comment above, posted 3 weeks ago, you said the following: "I think the links should use {project-context} ...". 
For this reason, I replaced all "admin" occurrences with {project-context}. 
Now you seem to be saying that I should do the reverse operation, i.e. replace {project-context} with "admin".
Can you please explain if there is something I am getting wrong?

Reply:
I also got confused by the other example which in fact uses `{project-context}` as context instead of a hardcoded, different value for each guide. Examples:

* [`master.adoc` in Tuning Foreman](https://github.com/theforeman/foreman-documentation/blame/master/guides/doc-Tuning_Performance/master.adoc#L3) uses `performance-tuning
` which is then "appended" to all links: https://docs.theforeman.org/nightly/Tuning_Performance/index-katello.html#Apache_HTTPD_Performance_Tuning_performance-tuning
* [`master.adoc` in Managing Content](https://github.com/theforeman/foreman-documentation/blame/master/guides/doc-Managing_Content/master.adoc#L3) uses `content-management` which results in links such as https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Creating_an_Activation_Key_to_Consume_the_Content_View_content-management

The "Administering Project" guide uses `admin`: https://github.com/theforeman/foreman-documentation/blame/master/guides/doc-Administering_Project/master.adoc#L3

IMHO there is only few guides that use `{project-context}` as context:

````
$ git grep "^:context:" **/master.adoc
guides/doc-Administering_Project/master.adoc::context: admin
guides/doc-Administering_with_Ansible/master.adoc::context: ansible-admin
guides/doc-Configuring_Load_Balancer/master.adoc::context: load-balancing
guides/doc-Deploying_Hosts_AppCentric/master.adoc::context: application_centric_deployment
guides/doc-Deploying_Project_on_AWS/master.adoc::context: deploying-on-aws
guides/doc-Installing_Proxy/master.adoc::context: {smart-proxy-context}
guides/doc-Installing_Server/master.adoc::context: {project-context}
guides/doc-Installing_Server_Disconnected/master.adoc::context: satellite
guides/doc-Managing_Configurations_Ansible/master.adoc::context: ansible
guides/doc-Managing_Configurations_Puppet/master.adoc::context: managing-configurations-puppet
guides/doc-Managing_Content/master.adoc::context: content-management
guides/doc-Managing_Hosts/master.adoc::context: managing-hosts
guides/doc-Managing_Organizations_and_Locations/master.adoc::context: managing-organizations-locations
guides/doc-Planning_for_Project/master.adoc::context: planning
guides/doc-Provisioning_Hosts/master.adoc::context: provisioning
guides/doc-Quickstart/master.adoc::context: {project-context}
guides/doc-Tuning_Performance/master.adoc::context: performance-tuning
guides/doc-Upgrading_and_Updating/master.adoc::context: upgrade-guide
````

For orcharhino, the downstream product of my employer, I don't use installing Server/Proxy and Quickstart guides, so I had less experience with them.

Long story short: if you link to the "Administering Project" guide using `{AdministeringDocURL}`, set the context to "admin".

Regarding cherry-picks: I've also ticked the box for 3.4, because this makes cherry-picked possible commits on top easier. Requesting cherry-picks to both 3.5 and 3.3 is almost always the case, unless we're documenting cutting edge things.

Reply:
Thank you for the explanation. I will check and set the correct context this evening.